heka/sandbox/filters/telemetry_decoder_view.lua

```diff
@@ -14,7 +14,9 @@ require "string"
 local SEC_PER_ROW   = 60
 local ROWS          = 2880
 
-bf                  = bloom_filter.new(3*1e6, 0.01) -- todo up the size when not under the sandbox manager
+local items         = read_config("bloom_items") or 3*1e6
```

Comment:
Altering these settings will require a preservation version to be added and since they are user configurable a little bit extra.
```lua
_PRESERVATION_VERSION = read_config("preservation_version") or 0
_PRESERVATION_VERSION = _PRESERVATION_VERSION + 1 -- force a revision update due to internal changes
```